src/app/util/DataModelHandler.cpp

```diff
@@ -22,20 +22,15 @@
 
 #include <app/util/DataModelHandler.h>
 
-#if __has_include(<zap-generated/endpoint_config.h>)
-#define USE_ZAP_CONFIG 1
 #include <app/util/attribute-storage.h>
 #include <app/util/util.h>
-#endif
 #include <lib/support/logging/CHIPLogging.h>
 
 using namespace ::chip;
 
 void InitDataModelHandler()
```

Comment:
This probably should soon be renamed `InitZapBasedDataModelHandler`

Reply:
Could you create an issue (or PR since simple sed) for this?

I think it would be indeed clearer on dependencies.